MQTTPacket/src/MQTTUnsubscribeServer.c

```diff
@@ -81,6 +81,8 @@ int32_t MQTTDeserialize_unsubscribe(unsigned char* dup, unsigned short* packetid
 	*count = 0;
 	while (curdata < enddata)
 	{
+		if (*count == maxcount)
```

Comment:
`>=` is safer.

Also, we'll need to scan all other APIs for possible similar issues.

Reply:
change to >=.But in MQTTV5Deserialize_subscribe,it use *count == maxcount to check.